src/assets/data/markdown/ionic-react/start.md

```diff
@@ -2,24 +2,59 @@
 
 In this lab, you will:
 
-- Start a new Ionic React application
-- Add some code formatting tooling
-- Restructure the starter for our base application
+- Use the CLI to create a starter application
+- Learn some Ionic CLI commands
 - Build and run the starter application
 
-## Start Here
+## Overview
 
-The Ionic CLI includes a command called `start` that we can use to bootstrap an application. The `start` command will use a basic starter application as its starting point. It is best to begin with the starter that most closely resembles the type of application we would like to create. In our case, we want a basic tabs based application, so we will use the `tabs` starter.
+THe Ionic command line is the main tool used to develop Ionic applications. This tool allows you to generate new applications, built the application, interact with Appflow and many other tasks. The command line also wraps and extends other command lines such as the Capacitor command line, react-scripts command line, etc.
+
+If you type `ionic --help` at the command line you get a list of the available top level commands that can be run via the CLI. These commands are broken into two sections: Global Commands and Project Commands. Global Commands can be run from anywhere where as the Project Commands can only be run from an Ionic project directory. Common top level commands include `start`, `info`, `generate`, and `serve`. We will learn more about these commands as we use them.
```

Comment:
> Common top level commands include `start`, `info`, `generate`, and `serve`.

The "top level" bit was kinda confusing when placed right after "Global" and "Project" commands, so I removed that in the Angular one. It now reads like this:

Commonly used commands include `start`, `info`, `generate`, and `serve`.

I don't know if `generate` makes sense in the React context. You may want to find another commonly used command.